packages/meditrak-server/src/routes/exportSurveys/cellBuilders/KeyValueCellBuilder.js

```diff
@@ -34,7 +34,7 @@ export class KeyValueCellBuilder {
     }
     const fullObject =
       typeof jsonStringOrObject === 'string' ? JSON.parse(jsonStringOrObject) : jsonStringOrObject;
-    const object = this.extractRelevantObject(fullObject);
+    const object = this.extractRelevantObject(fullObject) || {};
```

Comment:
@chris-bes I changed the behaviour in the parent class to make config for object fields optional. I believe this is a reasonable default.

If a config is required, child classes of `KeyValueCellBuider` could throw an explicit error in their implementation of the `extractRelevantObject` method. Even better, this should be caught up using validation during import (see eg [EntityConfigValidator](https://github.com/beyondessential/tupaia/blob/dev/packages/meditrak-server/src/routes/importSurveys/Validator/ConfigValidator/EntityConfigValidator.js)